v7/wordpress_compiler/wordpress/__init__.py

```diff
@@ -0,0 +1,24 @@
+# A WordPress compiler plugin for Nikola
+#
+# Copyright (C) 2014-2015 by the contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from __future__ import unicode_literals
+
+from .wordpress import CompileWordpress
+
+
+__all__ = ['CompileWordpress', 'WordPressPlugin']
```

Comment:
Thanks! I moved `WordPressPlugin` to `plugin_interface`, otherwise yapsy would (at least for me) load that definition as the plugin and ignore the page compiler...